src/main/scala/com/codacy/client/bitbucket/service/PullRequestServices.scala

```diff
@@ -68,15 +68,17 @@ class PullRequestServices(client: BitbucketClient) {
     client.postJson(Request(url, classOf[JsObject]), JsNull)
   }
 
-  def createComment(author: String, repo: String, prId: Int, commitUUID: String, body: String, file: Option[String], line: Option[Int]): RequestResponse[PullRequestComment] = {
```

Comment:
If you can, please create a new method for the comments, to avoid exposing the JsValue.